cmd/snap/cmd_changes.go

```diff
@@ -122,14 +125,55 @@ func (c *cmdChanges) Execute(args []string) error {
 
 func (c *cmdTasks) Execute([]string) error {
 	cli := Client()
-	return showChange(cli, c.Positional.ID)
+	var id changeID
+	if c.Positional.ID == "" && c.LastChangeType == "" {
+		return fmt.Errorf(i18n.G("please provide change ID or type with --last=<type>"))
+	}
+	if c.Positional.ID != "" && c.LastChangeType != "" {
+		return fmt.Errorf(i18n.G("change use ID and type together"))
+	}
+	if c.LastChangeType != "" {
+		kind := c.LastChangeType
+		// our internal change types use "-snap" postfix but let user skip it and use short form.
+		if kind == "refresh" || kind == "install" || kind == "remove" || kind == "connect" || kind == "disconnect" || kind == "configure" || kind == "try" {
+			kind += "-snap"
+		}
```

Comment:
this is not an exhaustive list of all kinds, just the ones for which kind == command+"-snap"

for example auto-refresh is an interesting one not of that form